### DISCUSSION TITLE
Quick Search For Shortcuts?

### DISCUSSION THREAD

**Discussion by egoldy** [Anze/KeyCluCask#93] [Q&A]
Is there an easy way to search all shortcuts and only display the ones you find?   Hot key the table, but then filter to specific shortcuts?

**Comment by Anze** [comment]
Hey [egoldy](https://github.com/egoldy),
Thanks for your suggestion! The feature you’re asking for is available. The filtering shortcuts functionality located at "Experimental Features" due to its unstable state. To access this feature, you'll need to [follow the instructions](https://github.com/Anze/KeyCluCask/wiki/Experimental-Features) provided to enable Experimental Features.

**Reply by dayyi** [reply]
Hello @Anze,
Is it possible to really filter the results instead of only highlighting them or this is not supported?

**Reply by Anze** [reply]
Technically, it is possible but not yet implemented. I'll aim to enhance the filtering functionality in future releases.
